src/DynamoCore/Graph/Nodes/NodeCategories.cs

```diff
@@ -420,8 +421,8 @@ internal static string MakeRelativePath(string basePath, string subjectPath)
             var documentUri = new Uri(basePath, UriKind.Absolute);
             var assemblyUri = new Uri(subjectPath, UriKind.Absolute);
 
-            var relativeUri = documentUri.MakeRelativeUri(assemblyUri);
-            var relativePath = relativeUri.OriginalString.Replace('/', '\\');
+            var relativeUri = documentUri.MakeRelativeUri(assemblyUri).OriginalString;
+            var relativePath = WebUtility.UrlDecode(relativeUri.Replace("+", "%2B")).Replace('/', '\\');
```

Comment:
"this is part of the DynamoCore.dll... so I am pretty sure it will get built by mono."

Oh, so it will still affect compilation?  I don't have enough time with C# to know for sure.  I can try a mono build.

"can you also explain the logic here ... not exactly sure what is going on here given that your description states url decoding is removed from display?"

There are two different but related issues here.
- The functions in Converters.cs are using UrlDecode on a string that is not url encoded.
- MakeRelativePath can improperly return a url encoded path.

MakeRelativePath should not return a URL encoded path.
new Uri() results in a URL encoded path.
Therefore, to undo that, we need to call UrlDecode on it.
Also, UrlDecode will convert + to space, but the Uri creation doesn't encode + as %2B.
In order to avoid + in the filename being converted to space, we need to encode + as %2B before calling it.

"Should a comment get added to this method summary about this change?"
I don't think there is a need for that.  This is just fixing a bug in the implementation of this function, and that bug is not currently mentioned in the method summary.

"Have you tried running the tests locally?"
As far as I know, there are no UI tests locally, and this is only called through the UI, so I don't know that this will get hit.  I have tested it manually.